css-three-column-layout/styles.css

```diff
@@ -0,0 +1,60 @@
+* {
+  font-family: Lato, sans-serif;
+  background-color: rgb(13 162 255);
+  color: white;
+}
+
+.container {
+  width: 1200px;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
```

Comment:
Instead of centering the stuff in the row, try just making your container narrower and then the columns each 1/3 wide instead of 1/5